content/en/docs/Tasks/Troubleshooting/subscription.md

```diff
@@ -30,15 +30,16 @@ If the `Status` block does not provide enough information, check the [Catalog op
 Each Subscription in a namespace acts as a part of a set of operators for the namespace - think of a Subscription as an entry in a python `requirements.txt`. If OLM is unable to resolve part of the set, it knows that resolving the entire set will fail, so it will bail out of the installation of operators for that particular namespace. Subscriptions are separate objects but within a namespace they are all synced and resolved together.
 
 ### Subscription fails due to missing APIs
-Newer versions of Kubernetes may deprecate and remove certain APIs that OLM supports. For example, the `apiextensions.k8s.io/v1beta1` GroupVersion is no longer available on Kubernetes 1.22+ clusters. Operators that rely on v1beta1 CRDs will fail to install on 1.22+ clusters, since the v1beta API has been removed. See 
-[the offical Kubernetes deprecation guide](https://kubernetes.io/docs/reference/using-api/deprecation-guide/) for the full list of resources that have been deprecated and removed by individual Kubernetes versions. These APIs will continue to be available on older versions of Kubernetes and OLM will continue to support them moving forward. 
 
-In the case where a subscription is created to an operator that relies on missing APIs, for example a v1beta1 CRD, an error condition will be present on the subscription status with the following message: 
+Newer versions of Kubernetes may deprecate and remove certain APIs that OLM supports. For example, the `apiextensions.k8s.io/v1beta1` GroupVersion is no longer available on Kubernetes 1.22+ clusters. Operators that rely on v1beta1 CRDs will fail to install on 1.22+ clusters, since the v1beta API has been removed. See
+[the official Kubernetes deprecation guide](https://kubernetes.io/docs/reference/using-api/deprecation-guide/) for the full list of resources that have been deprecated and removed by individual Kubernetes versions. These APIs will continue to be available on older versions of Kubernetes and OLM will continue to support them moving forward.
 
-```
+In the case where a subscription is created to an operator that relies on missing APIs, for example a v1beta1 CRD, an error condition will be present on the subscription status with the following message:
+
+```console
```

Comment:
what is the effect of this?

Reply:
It shouldn't have any user-facing effects - namely for silencing markdown warnings locally and slightly highlights the message more when looking at it through markdown preview.

Reply:
Updated to remove the change entirely.